lib.php

```diff
@@ -31,8 +31,9 @@
  * @return void
  * @throws \moodle_exception
  */
-function tool_mfa_after_require_login() {
-    global $SESSION, $ME, $CFG;
+function tool_mfa_after_require_login($courseorid = null, $autologinguest = null, $cm = null,
+    $setwantsurltome = null, $preventredirect = null) {
```

Comment:
I've never actually used $setwantsurltome but seems like we should honor this flag too?